website/src/pages/BlogPage/index.tsx

```diff
@@ -203,7 +203,8 @@ const BlogPage: React.FC = () => {
         ) : (
           " "
         )}
-
+        {currentBlogDetails ? 
```

Comment:
add this check to line no 125, same suggestion as above

Reply:
https://github.com/openebs/website/pull/172#discussion_r673953880